identity/src/main/java/com/stripe/android/identity/ui/DocSelectionScreen.kt

```diff
@@ -0,0 +1,198 @@
+package com.stripe.android.identity.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Divider
+import androidx.compose.material.LocalContentColor
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.identity.R
+import com.stripe.android.identity.navigation.DocSelectionFragment.Companion.SELECTION_NONE
+import com.stripe.android.identity.networking.Resource
+import com.stripe.android.identity.networking.Status
+import com.stripe.android.identity.networking.models.CollectedDataParam.Type
+import com.stripe.android.identity.networking.models.VerificationPage
+import com.stripe.android.identity.networking.models.VerificationPage.Companion.requireSelfie
+
+internal const val docSelectionTitleTag = "Title"
+internal const val singleSelectionTag = "SingleSelection"
+
+@Composable
+internal fun DocSelectionScreen(
+    verificationPageState: Resource<VerificationPage>,
+    onDocTypeSelected: (Type, Boolean) -> Unit
+) {
+    MdcTheme {
+        when (verificationPageState.status) {
+            Status.SUCCESS -> {
+                val documentSelect =
+                    remember { requireNotNull(verificationPageState.data).documentSelect }
+                val requireSelfie =
+                    remember { requireNotNull(verificationPageState.data).requireSelfie() }
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(
+                            start = dimensionResource(id = R.dimen.page_horizontal_margin),
+                            end = dimensionResource(id = R.dimen.page_horizontal_margin),
+                            top = dimensionResource(id = R.dimen.page_vertical_margin),
+                            bottom = dimensionResource(id = R.dimen.page_vertical_margin)
+                        )
```

Comment:
Nit: Minor suggestion.

```suggestion
                        .padding(
                            horizontal = dimensionResource(id = R.dimen.page_horizontal_margin),
                            vertical = dimensionResource(id = R.dimen.page_vertical_margin)
                        )
```